src/controllers/content.controller.ts

```
@@ -445,6 +445,12 @@ export class contentController {
     @Query() { limit = 5 },
   ) {
     try {
+      // Add the check for the limit
+      if(limit < 5){
```

ESLint

[error] Replace `(limit·<·5)` with `·(limit·<·5)·`

(prettier/prettier)